tools/include/endian.h

```diff
@@ -10,6 +10,14 @@
 #define bswap_16(x) OSSwapInt16(x)
 #define bswap_32(x) OSSwapInt32(x)
 #define bswap_64(x) OSSwapInt64(x)
+#define be16toh(x) OSSwapBigToHostInt16((x))
+#define htobe16(x) OSSwapHostToBigInt16((x))
+#define be32toh(x) OSSwapBigToHostInt32((x))
```

Comment:
Why do you have here `(( ))`? So double brackets?

Reply:
ping @lovehackintosh

Reply:
I just copied it from https://github.com/libressl/portable/blob/master/include/compat/endian.h#L54

Reply:
Removed double brackets, thank you.